packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx

```
@@ -1,179 +1,112 @@
-                                data-testid="@tooltip/guideAnchor"
-                                id="/1_initialize-and-secure-your-trezor/6_passphrase.md"
-                            />
-                        }
-                    />
-                </Column>
-            </CardWithDevice>
-        </SwitchDeviceModal>
+    const deviceOffer = !!(
+        device.features &&
+        device.features.capabilities &&
+        device.features.capabilities.includes('Capability_PassphraseEntry')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)